stablehlo/reference/Tensor.h

```diff
@@ -71,6 +71,8 @@ class Tensor {
   /// \name Constructors
   /// @{
   Tensor();
+  /// This constructor initializes the tensor with a scalar value.
+  explicit Tensor(const Element &initValue);
```

Comment:
I don't think this additional API brings much value at the moment. It's used only once, so I think we should just inline this constructor into the callsite, until we have more use cases for it.

Reply:
Oops the other changes weren't included. This is actually used by several other functions and improves the readability quite a bit. PTAL at the updated code.

Reply:
Agreed that this improves readability, but perhaps let move this to a helper function `makeScalar` next to `makeTensor`? While we're at it, perhaps let's change the `ShapedType type, const Element &initValue` constructor into a helper function as well, called `makeSplat` or something?